src/pages/import.html

```diff
@@ -48,40 +49,52 @@ <h1>
                                accept="{{fileFormatsExtended}}"
                                ngf-change="fileSelected($files, $file, $newFiles, $duplicateFiles, $invalidFiles, $event)"
                                ngf-max-size="maxUploadFileSizeMB + 'MB'">
-                                <em class="icon-upload icon-lg pull-left"></em>
-                                <div>
-                                    <span>{{'upload.rdf.files.label' | translate}}</span>
-                                    <br>
-                                    <small class="text-muted">{{'all.rdf.formats.label' | translate}}{{'up.to' | translate}} {{maxUploadFileSizeMB | number}} MB</small>
+                                <div class="grid-container">
+                                    <div class="text-and-icon">
```

Comment:
IMO this additional div is useless. You can just apply styles to the above wrapper

Reply:
Yes, took care of this